validator/testcases/javascript/predefinedentities.py

```diff
@@ -228,7 +228,7 @@ def is_shared_scope(traverser, right=None, node_right=None):
             '`InstallTrigger` API: '
             'https://developer.mozilla.org/en-US/docs/Web/API/InstallTrigger',
             'Updates to existing add-ons should be provided via the '
-            'install.rdf `updateURL` mechanism.'),
+            'manifest\'s `updateURL` mechanism.'),
```

Comment:
`"` over `'` for this? I know we switch to doubles elsewhere but it's weird with a multi-line.